File: desktop/src/com/github/servb/antonbird/desktop/DesktopLauncher.java
package com.github.servb.antonbird.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.github.servb.antonbird.AbGame;

public final class DesktopLauncher {

    public static void main(final String[] arg) {
        final LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
        config.title = "Anton Bird";
        config.width = 272;
        config.height = 408;
        new LwjglApplication(new AbGame(), config);
    }
}

File: desktop/src/com/github/servb/antonbird/desktop/DesktopLauncher.kt
package com.github.servb.antonbird.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.github.servb.antonbird.AbGame

object DesktopLauncher {

    @JvmStatic
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration().apply {
            title = "Anton Bird"
            width = 272
            height = 408
        }
        LwjglApplication(AbGame(), config)
    }
}
